scripts/run-dependencies

```diff
@@ -43,10 +43,51 @@ fi
 
 docker exec daedalus-web-app-db wait-for-db
 
-docker network create daedalus
+NETWORK=daedalus
+VOLUME=daedalus-results
+
+docker network create $NETWORK
 docker network connect daedalus daedalus-web-app-db
 
+docker volume create VOLUME
+
 # Pull and run the R API image
-R_API_IMAGE='mrcide/daedalus.api:latest'
+# TODO: reset this to latest when the rrq branch of the api is merged
+R_API_IMAGE='mrcide/daedalus.api:jidea-74'
 docker pull $R_API_IMAGE # There will be a 'latest' tag: https://github.com/jameel-institute/daedalus.api/pull/2#issuecomment-2252491603
-docker run -d --name daedalus-api --rm --network=daedalus -p 8001:8001 $R_API_IMAGE
+
+QUEUE_ID=daedalus-api-queue-$(uuidgen)
+
+docker run --rm -d \
+  -p 127.0.0.1:6379:6379 \
+  --name=daedalus-redis \
+  --network=$NETWORK \
+  redis
+
+# Ephemeral container for one-time queue configuration
```

Comment:
What makes it ephemeral? Does it get used once and hang around or do we remove it?

Reply:
Its entry point script is just doing one thing and then exiting so it stops  naturally  - unlike the api and the worker which wait to be given things to do. The `--rm` flag on the docker run means that it removes itself on exit.